core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dataprocModels.scala

```diff
@@ -103,4 +103,6 @@ object DataprocClusterStatus extends Enum[DataprocClusterStatus] {
   case object Unknown extends DataprocClusterStatus // The cluster state is unknown.
 
   case object Updating extends DataprocClusterStatus // The cluster is being updated. It continues to accept and process jobs.
+
+  case object Stopped extends DataprocClusterStatus // The cluster is being updated. It continues to accept and process jobs.
```

Comment:
Update comment

Reply:
good point...I haven't noticed it during test....will double check